src/main/java/seedu/duke/util/DateComparator.java

```diff
@@ -0,0 +1,12 @@
+package seedu.duke.util;
+
+import java.util.Comparator;
+import java.time.LocalDate;
+
+public class DateComparator implements Comparator<LocalDate> {
+
+    @Override
+    public int compare(LocalDate o1, LocalDate o2) {
+        return o1.compareTo(o2);
+    }
```

Comment:
Not to sure about this, but is there a need to create a class for this? Does implementing to a class, say Event or DateTimeManager and then overriding in that class work? Like wise for the DateTimeComparator, since a class can implements multiple interfaces.

Reply:
Er actually theres no significant need to do this as a class anymore. I was still learning on functional programming and functional interfaces in Java and their syntaxes so I implemented this as a stop-gap measure. Will update it.